index.ts

```diff
@@ -106,6 +107,11 @@ export default function d3scription<T> (contentGetter : ContentGetter<T>, option
 
                 return publicMethods;
             },
+            update(data : T) : Tip<T> {
+                tip.html(contentGetter(data));
+
+                return publicMethods;
+            },
```

Comment:
I wonder why is this update needed. Shouldn't be this be solved by updating data binded to the element in usual d3 way?

Reply:
Not sure what do you mean by that. We are using this as standalone for statCard tooltip in Elm. So it's wired via x-tag and we have access only to those public methods. But I need change content even if instance of `d3scription` tip is not changed. Content for it is pure HTML (string) so I do not have any more control over and calling `show` again cause issue with position of tooltip (because of internal `updateTipPosition();` call).

Reply:
Btw I have no access to be able publish this package, I'm trying to find, how can I change this, but so far no luck. 

```403 Forbidden - PUT https://registry.npmjs.org/d3scription - You do not have permission to publish "d3scription". Are you logged in as the correct user?```

So if you have the power, I created `jjelinek` account in npm to myself @turboMaCk  :)

Reply:
Right for usage without D3 based charts this could be handy. But even then I guess there mist be some element that binds data in d3 way so I think it should be possible to update tooltip data by binding new data to the element.

For the npm publish I think you'll need my assistance. I'll have a look how can I add you as a maintainer.

Reply:
@turboMaCk 😂 gonna send another invoice to GWI?

Reply:
Not sure if I would feel comfortable with taking this package hostage:D